libs/utils/utils.js

```diff
@@ -198,6 +206,34 @@ export function getLocale(locales, pathname = window.location.pathname) {
   return locale;
 }
 
+export function getLanguage(languages, locales, pathname = window.location.pathname) {
+  const defaultLang = { ietf: 'en', tk: 'hah7vzn.css', prefix: '/en' };
+  if (!languages) {
+    return defaultLang;
+  }
+  const split = pathname.split('/');
+  const baseSplit = [LANGSTORE, PREVIEW].includes(split[1]) ? 1 : 0;
```

Comment:
What's split, could we be more specific? `urlPath`? Similar, baseSplit might be more `locOffset` ?

Reply:
sure, the naming isn't great I agree. I'll rename 👍